rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -0,0 +1,66 @@
+import rclpy
```

Comment:
This is from the other PR, right?

Reply:
Yeah they're using the gameplay entry point branch so its easy for them to test